src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/OrganisationIdentifierValidatorImpl.java

```diff
@@ -3,18 +3,28 @@
 import java.util.Collection;
 
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.dao.EmptyResultDataAccessException;
 import org.springframework.security.access.AccessDeniedException;
 import org.springframework.security.core.GrantedAuthority;
 import org.springframework.stereotype.Component;
 import org.springframework.util.StringUtils;
+import uk.gov.hmcts.reform.professionalapi.controller.advice.ResourceNotFoundException;
 import uk.gov.hmcts.reform.professionalapi.domain.Organisation;
 import uk.gov.hmcts.reform.professionalapi.domain.OrganisationStatus;
+import uk.gov.hmcts.reform.professionalapi.service.OrganisationService;
 import uk.gov.hmcts.reform.professionalapi.util.RefDataUtil;
 
 @Component
 @Slf4j
-public class OrganisationIdentifierIdentifierValidatorImpl implements OrganisationIdentifierValidator {
+public class OrganisationIdentifierValidatorImpl implements OrganisationIdentifierValidator {
+
+    private OrganisationService organisationService;
+
+    @Autowired
+    public OrganisationIdentifierValidatorImpl(OrganisationService organisationService) {
+        this.organisationService = organisationService;
+    }
```

Comment:
remove service from here.

Reply:
I need the Service. I had repository in here, but you said to replace that with a Service call...